documentation/quarkus/guide-authentication-quarkus.asciidoc

```diff
@@ -0,0 +1,17 @@
+:toc: macro
+toc::[]
+
+= Quarkus Authentication
+Quarkus supports different authentication mechanisms through different extensions. For example:
+
+* https://quarkus.io/guides/security-properties[`quarkus-elytron-security-properties-file`] provides support for simple properties files that can be used for testing security. This supports both embedding user information in `application.properties` and standalone properties files.
+* https://quarkus.io/guides/security-jpa[`quarkus-security-jpa`] provides support for authenticating via JPA.
+* https://quarkus.io/guides/security-jdbc[`quarkus-elytron-security-jdbc`] provides support for authenticating with JDBC
+* https://quarkus.io/guides/security-jwt[`quarkus-smallrye-jwt`] provides support for authenticating using JSON Web Tokens. It allows you to inject the token and claims into the application as per the MicroProfile JWT specification.
+* https://quarkus.io/guides/security-openid-connect[`quarkus-oidc`] provides support for authentication via an OpenID Connect provider e.g. KeyCloak
+* https://quarkus.io/guides/security-keycloak-authorization[`quarkus-keycloak-authorization`] provides support for a policy enforcer using https://www.keycloak.org/docs/latest/authorization_services/index.html[Keycloak Authorization Services].
+* ...
```

Comment:
better omit this or explicitly write something like "For additional options search on https://quarkus.io/...[quarkus]".